portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,35 +26,63 @@
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
 import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.sps.data.Comment;
 
-
-/** Servlet that returns some example content. */
+/** Servlet that returns data for my portfolio. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  private List comments = new ArrayList<String>();
+  private List comments = new ArrayList<Comment>();
```

Comment:
Why is this a global variable? Why isn't it function-scoped?

Reply:
you are correct. before I used datastore it was used in both functions, but now only in one. Done :) 
should the:
>private static final int DEFAULT_NUM_COMMENTS = 10;
also go inside the function? I didn't find a reference in the style guide to the scope finals should be in.

Reply:
https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

There's no reference to global constant. However, it's common practice to have constant values as `private static final` class members.

Reply:
great. thanks